core/modules/user/user.pages.inc

```diff
@@ -51,6 +51,9 @@ function user_pass() {
     '#default_value' => isset($_GET['name']) ? $_GET['name'] : '',
     '#attributes' => array(
       'autofocus' => 'autofocus',
+      'autocapitalize' => 'none',
+      'autocorrect' => 'off',
+      'spellcheck' => 'false',
```

Comment:
This block simply adds the desired attributes to both name and password for this specific high-level form.